xtas/tasks/single.py

```diff
@@ -30,7 +30,7 @@
 
 from cytoolz import identity, pipe
 import nltk
-import spotlight
```

Comment:
Why are nltk and spotlight imported globally, and e.g. frog locally? Especially spotlight is only used in one function.